client/settings/dpcs-settingspanel/panel_evoker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ "$BASH_SOURCE" == "$0" ]]
+then 
+	sudo -s python2 settingspanel.py
```

Comment:
I guess that Synaptics might use `gksudo` or sth similar. Still it should be used like `gksudo ./panel_evoker.sh` instead of hard-coding into the script.